webapp/channels/src/components/team_controller/actions/index.ts

```diff
@@ -30,14 +29,6 @@ export function initializeTeam(team: Team): ActionFuncAsync<Team> {
         const currentUser = getCurrentUser(state);
         LocalStorageStore.setPreviousTeamId(currentUser.id, team.id);
 
-        try {
-            await dispatch(fetchChannelsAndMembers(team.id));
```

Comment:
`initializeTeam` is also called in `joinTeam` below here for when you join a new team. We probably want to fetch channels and members in there, so could you re-add this to `joinTeam`?

Reply:
Oh good one. Will do.